VirtualList/tests/useEvent-translate-specs.js

```diff
@@ -0,0 +1,378 @@
+			`%3Ctext x='50%25' y='50%25' dominant-baseline='middle' text-anchor='middle' font-size='36px' fill='%23${textColor}'%3E${customText}%3C/text%3E%3C/svg%3E`
+		);
+
+		for (let i = 0; i < dataSize; i++) {
+			const color = Math.floor(Math.random() * (0x1000000 - 0x101010) + 0x101010).toString(16);
+
+			items.push({
+				name: 'Account ' + i,
+				// For VirtualGridList
+				subText: 'SubText ' + i,
+				source: svgGenerator(300, 300, color, 'ffffff', `Image ${i}`)
+			});
+		}
+	});
+
+	afterEach(() => {
+		clientSize = null;
+		dataSize = null;
+		imageItemSize = null;
+		items = [];
+		itemSize = 60;
+		items = null;
+		renderImageItem = null;
+		renderItem = null;
+		svgGenerator = null;
+	});
+
+	describe('VirtualList useEvent', () => {
+		test('should navigate focus using arrow-up/down key', () => {
+			render(
+				<VirtualList
+					clientSize={clientSize}
+					dataSize={dataSize}
+					itemRenderer={renderItem}
+					itemSize={itemSize}
+					scrollMode="translate"
+				/>
+			);
+
+			const list = screen.getByRole('list');
+			const item0 = list.children.item(0).children.item(0);
+			const item1 = list.children.item(1).children.item(0);
+			const item2 = list.children.item(2).children.item(0);
+
+			focus(item0);
+			expect(currentFocusIndex).toBe(0);
+
+			downKey(item0);
+			expect(currentFocusIndex).toBe(1);
+
+			downKey(item1);
+			expect(currentFocusIndex).toBe(2);
+
+			upKey(item2);
+			expect(currentFocusIndex).toBe(1);
+		});
+
+		test('should not navigate focus using arrow-up/down key when \'direction\'is horizontal', () => {
+			render(
+				<VirtualList
+					direction="horizontal"
+					clientSize={clientSize}
+					dataSize={dataSize}
+					itemRenderer={renderItem}
+					itemSize={itemSize}
+					scrollMode="translate"
+				/>
+			);
+
+			const list = screen.getByRole('list');
+			const item0 = list.children.item(0).children.item(0);
+
+			focus(item0);
+			expect(currentFocusIndex).toBe(0);
+
+			downKey(item0);
+			expect(currentFocusIndex).toBe(0);
+
+			upKey(item0);
+			expect(currentFocusIndex).toBe(0);
+		});
+
+		test('should navigate focus using arrow-left/right key when \'direction\'is horizontal', () => {
+			render(
+				<VirtualList
+					direction="horizontal"
+					clientSize={clientSize}
+					dataSize={dataSize}
+					itemRenderer={renderItem}
+					itemSize={itemSize}
+					scrollMode="translate"
+				/>
+			);
+
+			const list = screen.getByRole('list');
+			const item0 = list.children.item(0).children.item(0);
+			const item1 = list.children.item(1).children.item(0);
+			const item2 = list.children.item(2).children.item(0);
+
+			focus(item0);
+			expect(currentFocusIndex).toBe(0);
+
+			rightKey(item0);
+			expect(currentFocusIndex).toBe(1);
+
+			rightKey(item1);
+			expect(currentFocusIndex).toBe(2);
+
+			leftKey(item2);
+			expect(currentFocusIndex).toBe(1);
+		});
+
+		test('should be scroll by focus navigation using arrow-down key', () => {
```

Comment:
```suggestion
		test('should scroll by focus navigation using arrow-down key', () => {
```
Please fix them all.